OPHD/Things/Structures/Residence.h

```diff
@@ -5,6 +5,9 @@
 #include <algorithm>
 
 
+const int ResidentialWasteCapacity = 1000;
```

Comment:
I thought I did that? Maybe I'm thinking the recycling facility.

Anyway, I would agree that this belongs in the class itself. That was a brainfart.

I keep forgetting constexpr is a thing. Since the address of the variable is moot (it's never referenced or pointed at, only used as a named constant), constexpr makes sense here as previously `#define` directives were the norm.